main.go

```diff
@@ -45,9 +46,14 @@ func setSwaggerInfo() {
 	docs.SwaggerInfo.Title = "helpwave rest-api"
 	docs.SwaggerInfo.Description = "helpwave rest-api backend"
 	docs.SwaggerInfo.Version = Version
-	docs.SwaggerInfo.Host = "main.helpwave.de"
+	docs.SwaggerInfo.Host = util.GetEnvOr("BASE_URL", "api.helpwave.de")
 	docs.SwaggerInfo.BasePath = "/v1"
-	docs.SwaggerInfo.Schemes = []string{"https"}
+
+	scheme := "https"
+	if util.GetEnvOr("UNSECURE", "disabled") == "enabled" {
```

Comment:
I'm not sure if this is needed, we should never accept http traffic, the swagger docs should reflect that.

Reply:
I think its important for local testing.
In production this option is disabled by default.
So there is no threat at all!

Reply:
Its acutally not part of the swagger docs.

Reply:
Im aware of this being out of scope but it was a great option to test the application using [http://localhost:3000/swagger/index.html](http://localhost:3000/swagger/index.html).

Reply:
My understanding is it only provides information for visitors of `/swagger/ index.html`. This way the frontend folks know what host to use. As we never use the unsecure host, I think this will cause confusion.

Reply:
But why? We ONLY support `http` instead of `https` explicitly using the `UNSECURE` env variable. So while the frontend folks would look up the api specs they will never get in touch with this variable. So its still just `https` on `api.helpwave.de`. Why is this confusing?
Also every production instance is using the default varibles there causing the swagger docs being rendered like before.
The only benefit is that we can acutally use this swagger feature set in development.
Also I think this swagger info manipulation based on the current environment is intended behavior.
Else you cant deploy this software on more than one host cause its locked to one static fixed api hostname and scheme.